Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -1097,6 +1097,26 @@
       components:
       - SurplusBundle
 
+- type: listing
+  id: UplinkStarterKit
+  name: uplink-starter-kit-name
+  description: uplink-starter-kit-desc
+  productEntity: ClothingBackpackDuffelSyndicateFilledStarterKit
+  cost:
+    Telecrystal: 40
+  categories:
+  - UplinkDisruption
+  conditions:
+    - !type:StoreWhitelistCondition
+      whitelist:
+        tags:
+          - NukeOpsUplink
+    - !type:BuyerWhitelistCondition
+      blacklist:
+        components:
+          - SurplusBundle
```

```suggestion
        - NukeOpsUplink
    - !type:BuyerWhitelistCondition
      blacklist:
        components:
        - SurplusBundle
```
[yaml convention](https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html#yaml-conventions)